packages/forms-web-app/src/dynamic-forms/questions.js

```diff
@@ -222,42 +196,6 @@ exports.questions = {
 		html: 'resources/notified-who/content.html',
 		documentType: documentTypes.whoWasNotified
 	}),
-	displaySiteNotice: new BooleanQuestion({
```

Comment:
these questions aren't used anywhere, they were replaced with howYouNotifiedPeople + the related uploads
have removed as they can cause confusion